sympy/tensor/index_methods.py

```diff
@@ -24,6 +23,7 @@ class IndexConformanceException(Exception):
     pass
 
 
+@deprecated(useinstead='EinsteinSum', issue=9284)
 def _remove_repeated(inds):
```

Comment:
I removed deprecation of private methods and added `index_methods` to the doctest blacklist. Not sure of a better way to do this since SymPy seems built around not testing deprecated code (since the deprecation warning counts as an error when testing).